packages/filebrowser-share-file/package.json

```diff
@@ -22,7 +22,7 @@
   "style": "style/index.css",
   "repository": {
     "type": "git",
-    "url": "https://github.com/tubitv/jupyterlab-extensions.git"
+    "url": "https://github.com/Tubitv/jupyterlab-extensions.git"
```

Comment:
github is case-sensitive?

Reply:
No, both `https://github.com/tubitv/jupyterlab-extensions/pull/2` and `https://github.com/Tubitv/jupyterlab-extensions/pull/2` point to this PR, but looks `Tubitv` is in its default one.